### ISSUE TITLE
🌐 Website Issue | Compliance with Web Content Accessibility Guidelines (WCAB)

### ISSUE THREAD

**Issue by Mikaela** [privacytoolsIO/privacytools.io#1809]
I think the website should be checked for compliance with WCAG and this could be a tracking issue for issues arising.

* https://www.w3.org/WAI/standards-guidelines/wcag/glance/

Currently known issues:

* https://github.com/privacytoolsIO/privacytools.io/issues/1471 - screen reader accessibility of icons
* https://github.com/privacytoolsIO/privacytools.io/issues/1808 - problems of visually impaired people
